luke-lukecongdon/hw_1/madlib-with-loops/js/madlib-console.js

```diff
@@ -0,0 +1,7 @@
+var startupX = ['Uber', 'Google', 'Amazon', 'Apple', 'Facebook', 'Twitter'];
+var startupY = ['Slack', 'Trello', 'Tesla', 'Hyperloop', 'Harvest'];
+
+var random1 = Math.floor((Math.random() * startupX.length));
+var random2 = Math.floor((Math.random() * startupY.length));
+
+console.log('A startup that is ' + startupX[random1] + ', but for ' + startupY[random2]);
```

Comment:
You just got this working today during office hours - I'd love to see the solution when you're ready to push it!